rs/rust_canisters/load_simulator/README.md

```diff
@@ -0,0 +1,15 @@
+Load Simulator Canister
+=======================
```

Comment:
Is this canister only intended for testing? If so, I recommend mentioning that in the readme (not necessarily in this PR).

Reply:
Added.